go.mod

```diff
@@ -1,30 +1,35 @@
 module github.com/docker/buildkit-syft-scanner
 
-go 1.20
+go 1.21.0
+
+toolchain go1.21.3
```

Comment:
I *think* so: https://go.dev/doc/modules/gomod-ref#go-notes (emphasis mine)

> Declares a suggested Go toolchain to use with this module. Only takes effect when the module is the main module and **the default toolchain is older than the suggested toolchain**.

So this was also auto-generated by `go mod tidy`. I had `go 1.21` written in the file, and `go 1.21.3` installed locally. `go mod tidy` tidies this to `go 1.21.0` and `toolchain go1.21.3`. I've bumped to use `go 1.21.3` so no toolchain directive is required.

Not quite sure what we want to do across the buildkit-related projects, looks like this is going to be the first of *many* cases.

Reply:
*oh* so `go mod tidy` is tidying to this because that's what syft specifies: https://github.com/anchore/syft/pull/2108.

Reply:
> _oh_ so `go mod tidy` is tidying to this because that's what syft specifies: [anchore/syft#2108](https://github.com/anchore/syft/pull/2108).

Oh boy :see_no_evil: